src/solv/model.f90

```diff
@@ -376,6 +382,25 @@ subroutine loadInternalParam(self, env, solvent, level)
             param = gfn1_alpb_hexane
          end select
       end if
+      
+      if (self%cosmo) then
+         if (verify(solvent,'0123456789.') .eq. 0) then
+            if (index(solvent,'.') .eq. index(solvent,'.',back=.true.)) then
```

Comment:
Yeah, this is meant to be a failsafe for a string with too many floating points (e.g. "78.3.6"). It will just fall through and trigger the "not parametrized" error.

Now that I think about it, it probably won't catch the edge case "--cosmo ."